airbyte-integrations/connectors/source-shortio/source_shortio/manifest.yaml

```diff
@@ -31,7 +31,7 @@ definitions:
     type: "DefaultPaginator"
     pagination_strategy:
       type: "CursorPagination"
-      cursor_value: "{{ last_records['nextPageToken'] }}"
```

Comment:
do we know if this even worked? I'd expect `last_records` to be an array, not an object